library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -65,6 +68,8 @@
     private CalendarDay currentMonth;
     private TitleFormatter titleFormatter = DEFAULT_TITLE_FORMATTER;
 
+    private Map<String,DayViewDecorator> dayViewDecorators;
```

Comment:
This approach simplifies decorator management, and even allow developer to dynamically update decorated days. I like it.